integration/snapshot_test.go

```diff
@@ -24,11 +24,11 @@ func TestSnapshot(t *testing.T) {
 	end := time.Now().Unix()
 	start := end - 3600
 
-	query_s := "avg:system.mem.used{*}"
-	url, err := client.Snapshot(query_s, time.Unix(start, 0), time.Unix(end, 0), "")
+	queryS := "avg:system.mem.used{*}"
```

Comment:
Might as well make this `queryString` or `q` as it is very short lived. I prefer `queryString` but Go devs seem to prefer short rather than long if the variable is short lived. See https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
I can make it q or perhaps even qs.